core/node/node_framework/src/implementations/layers/postgres.rs

```diff
@@ -0,0 +1,186 @@
+use std::time::Duration;
+
+use serde::{Deserialize, Serialize};
+use tokio::sync::watch;
+use zksync_dal::{
+    metrics::PostgresMetrics, system_dal::DatabaseMigration, ConnectionPool, Core, CoreDal,
+};
+use zksync_health_check::{Health, HealthStatus, HealthUpdater, ReactiveHealthCheck};
+
+use crate::{
+    implementations::resources::{
+        healthcheck::AppHealthCheckResource,
+        pools::{PoolResource, ReplicaPool},
+    },
+    service::StopReceiver,
+    task::{Task, TaskId, TaskKind},
+    wiring_layer::{WiringError, WiringLayer},
+    FromContext, IntoContext,
+};
+
+const SCRAPE_INTERVAL: Duration = Duration::from_secs(60);
+
+#[derive(Debug, Deserialize, Clone, PartialEq)]
+pub struct Config {
```

Comment:
This config looks redundant; it's not used and is always initialized to a default value. Also, you can probably reuse `SCRAPE_INTERVAL` for the new task.

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3193/commits/19275b6681b6a1ff7fe84f6a11a088d6d4a137f2